plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -238,6 +241,159 @@ export async function getOrganizationTeams(
   return { groups };
 }
 
+export async function getOrganizationTeamsFromUsers(
+  client: typeof graphql,
+  org: string,
+  userLogins: string[],
+  teamTransformer: TeamTransformer = defaultOrganizationTeamTransformer,
+): Promise<{
+  groups: GroupEntity[];
+}> {
+  const query = `
+   query teams($org: String!, $cursor: String, $userLogins: [String!] = "") {
+  organization(login: $org) {
+    teams(first: 100, after: $cursor, userLogins: $userLogins) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      nodes {
+        slug
+        combinedSlug
+        name
+        description
+        avatarUrl
+        editTeamUrl
+        parentTeam {
+          slug
+        }
+        members(first: 100, membership: IMMEDIATE) {
+          pageInfo {
+            hasNextPage
+          }
+          nodes {
+            login
```

Comment:
Are we able to fetch the rest of the member fields here as well? Same in the query below

Reply:
Done :)